clients/identity/authentication.go

```diff
@@ -101,6 +112,37 @@ func (api IdentityClient) CheckRequest(req *http.Request) (context.Context, int,
 	return ctx, http.StatusOK, nil
 }
 
+func splitTokens(florenceToken, authToken string) log.Data {
+	var ft, at tokenObject
+	logData := log.Data{}
+
+	if len(florenceToken) > 0 {
+		ft = splitToken(florenceToken)
+		logData["florence_token"] = ft
+	}
+
+	if len(authToken) > 0 {
+		at = splitToken(authToken)
+		logData["auth_token"] = at
+	}
+
+	return logData
+}
+
+func splitToken(token string) (tokenObj tokenObject) {
+
+	splitToken := strings.Split(token, " ")
+	tokenObj.numberOfParts = len(splitToken)
+	tokenObj.hasPrefix = strings.HasPrefix(token, common.BearerPrefix)
+	if len(token) > 6 {
+		tokenObj.tokenPart = token[len(token)-6:]
+	} else {
+		tokenObj.tokenPart = token
```

Comment:
am i reading this right that where we have a short token we would now be logging the whole thing?  i cant think of any cases where we'd have such short tokens but if we did i dont think its ever gonna be a good idea to log a full token 🤔